it-score.html

```diff
@@ -0,0 +1,121 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="/CSS_Components/styles.css">
+
+    <link rel="stylesheet" href="/CSS_Components/result.css">
+
+    <title>The Conjuring Quiz</title>
+</head>
+
+<body>
+    <!-- header container -->
+    <div class="header-content">
+        TRIVIA WITH TUSHAR
+    </div>
+
+    <!-- Quiz Container -->
+    <div class="quiz-content">
+        <div class="question">
+            <h3><span class="question-num">Question 1.</span> Who plays Richie Tozier in the movie?</h3>
+
+            <button class="answer">
+                Joe Keery
+            </button>
+
+            <button class="answer green">
+                Flinn Wolfhard
+            </button>
+
+            <button class="answer">
+                Jack Dylan Grazer
+            </button>
+
+            Correct answer ! Score: 10
+        </div>
+
+        <div class="question">
+            <h3><span class="question-num">Question 2.</span> Who became catatonic after getting inside "IT's" mouth?</h3>
+
+            <button class="answer">
+                Bill
+            </button>
+
+            <button class="answer green">
+                Beverly
+            </button>
+
+            <button class="answer">
+                Ben
+            </button>
+
+            Correct Answer ! Score : 10
+        </div>
+
+        <div class="question">
+            <h3><span class="question-num">Question 3.</span> Who killed Georgie?</h3>
+
+            <button class="answer">
+                Bill
+            </button>
+
+            <button class="answer">
+                Ben
+            </button>
+
+            <button class="answer green">
+                Pennywise
+            </button>
+
+            Correct answer ! Score: 10
+        </div>
+
+        <div class="question">
+            <h3><span class="question-num">Question 4.</span> Who is the leader of "The Loser's Club?</h3>
+
+            <button class="answer red">
+                Stanley Uris
+            </button>
+
+            <button class="answer">
+                Beverly Marsh
+            </button>
+
+            <button class="answer green">
+                Bill Denbrough
+            </button>
+
+            Oops! Wrong answer. Score : 0
+        </div>
+
+        <div class="question">
+            <h3><span class="question-num">Question 5.</span> After how many years does Pennywise the Dancing Clown wake up from his sleep?</h3>
+
+            <button class="answer green">
+                Every 27 years
+            </button>
+
+            <button class="answer red">
+                Every 10 years
+            </button>
+
+            <button class="answer">
+                Every 17 years
+            </button>
+
+            Oops! Wrong answer. Score : 0
+        </div>
+        
+        <div class="total-score">
+            Total Score: 30
+        </div>
+
+    </div>
+    </div>
```

Comment:
You have given the same answers to the questions of the conjuring quiz in it quiz